Remove old link references to docs from Community Overview page (#245)
mattermost/mattermost-handbook: contributors/contributors/community.md

Interested in contributing to Mattermost? [Learn about all the ways to get involved with our community](https://mattermost.com/contribute/).
{% endhint %}

### Users

> In general, all members of the Mattermost community are users of the open source project. They may be end users on a system, administrators, or developers on the Mattermost platform in some way, and the common bond we all share is we benefit from the project’s continued and growing success.

Reviewer: ```suggestion
In general, all members of the Mattermost community are users of the open source project. They can range from end users and system administrators, to developers on the Mattermost platform in some way. The common bond we all share is that we benefit from the project’s continued and growing success.
```

The user community has a number of segments:

#### End Users

People using Mattermost to communicate and achieve their organization’s top priorities.

#### IT Administrators

People deploying Mattermost and supporting end users.

#### System Integrators

> People who extend and customize Mattermost privately but who do not share back their work with the open source community are considered “users”. When the same people decide to share their work with others in the community they are recognized as “contributors”

Reviewer: ```suggestion
People who extend and customize Mattermost privately but who do not share back their work with the open source community (for a range of reasons, often related to confidentiality) are considered “users”. When users are able to share their work with others in the community they are recognized as “contributors”.
```

#### Customers

> Organizations that purchase Mattermost Enterprise Edition as an extension of the Mattermost open source project for advanced features. While the Mattermost roadmap is focused on benefiting the broad community, large customers may accelerate certain parts of the roadmap they find most valuable through large purchases of Enterprise Edition licenses.

Reviewer: ```suggestion
Organizations that purchase Mattermost Enterprise Edition as an extension of the Mattermost open source project for advanced features are known as "customers". While the Mattermost roadmap is focused on benefiting the broad community, large customers may accelerate certain parts of the roadmap they find most valuable through purchasing large numbers of Enterprise Edition licenses.
```

### Contributors

> A subset of the Mattermost community as “contributors” who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas or reporting bugs, offering professional services to install or deploy the system, or it can be as involved as building whole new products based on Mattermost, or contributing code to the underlying software.

Reviewer: ```suggestion
Contributors are a subset of the Mattermost community who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas, reporting bugs, or offering professional installation and deployment services. Or it can be as complex as building whole new products based on Mattermost, or contributing code to the underlying software.
```

> A subset of the Mattermost community as “contributors” who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas or reporting bugs, offering professional services to install or deploy the system, or it can be as involved as building whole new products based on Mattermost, or contributing code to the underlying software.

Reviewer: ```suggestion
A subset of the Mattermost community are known as “contributors” who help increase the overall benefits of Mattermost for others. This may be as simple as offering feature ideas or reporting bugs, offering professional services to install or deploy the system, or it can be as involved as building whole new products based on Mattermost, or contributing code to the underlying software.
```

> The work of all contributors is valued, appreciated and recognized as a driving force behind the project’s growth.

Reviewer: ```suggestion
The work of all contributors is valued, appreciated, and recognized as a driving force behind the project’s growth.
```

There are many ways to contribute:

#### Solution Contributors

> As of April 2019, [over 1,800 open source projects on GitHub reference Mattermost](https://github.com/search?q=mattermost). Solution Builders create a host of technologies to empower Mattermost software to solve real world problems. Technologies include:

Reviewer: ```suggestion
As of May 2020, [over 1,900 open source projects on GitHub reference Mattermost](https://github.com/search?q=mattermost). Solution Contributors create a host of technologies to empower Mattermost software to solve real world problems. Technologies include:
```

> Other contributor types beyond Solution Contributors are welcome to use and modify this LinkedIn template to document their contributions as well.

Reviewer: ```suggestion
Other contributor types beyond Solution Contributors are also welcome to use and modify this LinkedIn template to document their contributions.
```

Community Coordinator: Jason Blais and Ben Schumacher

#### Translation Contributors

Translation Contributors can create and update translations to existing or new Mattermost languages at [https://translate.mattermost.com](https://translate.mattermost.com/). In addition, community members can become “Language owners” and help review, coordinate and supervise the translation of Mattermost into languages they speak.

Community Coordinator: Jason Blais

#### Documentation Contributors

Documentation Contributors update, edit, and create new documentation, help text, training, and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).

Community Coordinator: Jason Blais and Justine Geffen

#### Quality Assurance Contributors

> Quality Assurance Contributors help find, confirm and report product defects via our process at [https://www.mattermost.org/filing-issues/](https://www.mattermost.org/filing-issues/). Anyone who encounters a verifiable defect is welcome to file a bug on the system.

Reviewer: Confirming with the QA team that this is the most up to date link for filing bugs.

Reviewer: Wondering if we should link to the Bugs channel too?

Author: We could link to the Bugs channel, but it's behind email login credentials -- so in that regard GitHub issues is often simpler for QA contributors.

Reviewer: That makes sense, we can disregard this suggestion. QA have confirmed that the contents in the link provided are correct, so resolving this comment.

> Bonus: When QA Contributor finds and reports a bug on a release candidate build of Mattermost prior to the final release of the system, the can be awarded a [Mattermost Bug Hunter Award](https://forum.mattermost.org/t/mattermost-bug-hunter-awards/4979) for their contribution.

Reviewer: ```suggestion
Bonus: When a QA Contributor finds and reports a bug on a release candidate build of Mattermost prior to the final release of the system, they can be awarded a [Mattermost Bug Hunter Award](https://forum.mattermost.org/t/mattermost-bug-hunter-awards/4979) for their contribution.
```

Community Coordinator: TBD

#### Knowledge Base Contributors

> Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).

Reviewer: ```suggestion
Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [troubleshooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and on the [Mattermost community server](https://community.mattermost.com/).
```

Community Coordinator: Amy Blais

#### Deployment Solution Contributors

> Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.

Reviewer: ```suggestion
Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build in line with Mattermost quality standards.
```

Reviewer: Wondering if a link to the SA channel or other relevant channel would be useful? Might also be a good jumping off point for communication between contributor and Mattermost.

Author: Good idea! Is the SA channel public?

Reviewer: Ah, good point. It's not. :(

Community Coordinator: Jason Blais and Amy Blais

#### UX Design Contributors

> UX Design Contributors influence the design of the Mattermost user experience through discussion in the `ux-design` channel on the Mattermost continous integration server at [https://community.mattermost.com/core/channels/ux-design](https://community.mattermost.com/core/channels/ux-design) and through contributions to the [Mattermost User Experience Guidelines](https://docs.mattermost.com/developer/fx-guidelines.html) on [GitHub](https://github.com/mattermost/docs/blob/master/source/developer/fx-guidelines.rst).

Reviewer: @andrewbrown00, is this link still good to use or should we link to the new design website?
